### PR TITLE
Fix website in microsoft edge via fetch-headers polyfill

### LINKED ISSUE #255: Website fails to load on Microsoft Edge
Using versions: 

Microsoft Edge 42.17134.1.0
Microsoft EdgeHTML 17.17134

When browsing to members.codefordenver.org, a plain white screen is displayed without any content. 
An error is printed to the console: `SCRIPT65535: Invalid argument`

### PR DESCRIPTION
**Pull request by dchao19**
This PR closes #255 

#### What does this PR do?
This PR adds the fetch-headers polyfill to fix Apollo-link-rest in Microsoft Edge. This isn't an ideal solution because it polyfills regardless of browser support, but Microsoft Edge has the wrong fetch-headers and it is incompatible with Apollo-link-rest. For more details: see https://github.com/apollographql/apollo-link-rest/issues/93.

Importantly, however, it fixes the website in Microsoft Edge.

### CODE REVIEW

File: src/config/polyfills.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+global.Headers = require('fetch-headers');
```
Comment (wdoug) [inline src/config/polyfills.js]: It's a little odd to me to be assigning this to `global` and just using the fact that our webpack setup has aliased `global` to `window` to be compatible with node

**Comment by wdoug** [comment]
I'm going to go ahead and rebase this to fix the package-lock.json conflict and then merge it

**Comment by wdoug** [comment]
I updated this branch, and I also submitted a pull request against the fetch-headers library to fix a few issues with it: https://github.com/jimmywarting/fetch-headers/pull/4

I think we should use [this polyfill](https://github.com/github/fetch) instead and just use their implementation of the Headers class.

File: src/config/polyfills.js (hunk 1 above)
Comment (dchao19) [reply src/config/polyfills.js]: Switching the polyfill to whatwg-fetch resolves this comment.

**Comment by dchao19** [comment]
You're right. I've updated this PR to use the whatwg-fetch polyfill and the fix appears to have stuck on Edge with versions:

Microsoft Edge 42.17134.1.0
Microsoft EdgeHTML 17.17134

**Review by wdoug — APPROVED** [review]
LGTM

**Pull request merged**